src/ui/utils/telemetry.ts

```diff
@@ -34,11 +34,20 @@ export function setupTelemetry(context: Record<string, any>) {
   Sentry.setContext("recording", { ...context, url: window.location.href });
 }
 
+type TelemetryUser = {
+  userId: string | undefined;
+  userEmail: string | undefined;
+  isInternal: boolean;
+};
+
+let telemetryUser: TelemetryUser;
```

Comment:
should this be `let telemetryUser: ?TelemetryUser;` since it will start life as undefined